### PR TITLE
Add column for merge_head, stop assuming it's always master

### PR DESCRIPTION
**Pull request by bdunne**

### CODE REVIEW

**Comment by bdunne** [comment]
@Fryguy Please review

File: db/migrate/20160119143158_add_merge_head_to_branch.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddMergeHeadToBranch < ActiveRecord::Migration
+  def change
+    add_column :branches, :merge_head, :string
+  end
```
Comment (Fryguy) [inline db/migrate/20160119143158_add_merge_head_to_branch.rb]: Needs a data migration to add a default of "master"

File: app/models/branch.rb [hunk 2]
```diff
@@ -43,6 +43,10 @@ def last_commit_uri
     commit_uri_to(last_commit)
   end
 
+  def local_merge_head
```
Comment (Fryguy) [inline app/models/branch.rb]: Something about the method name bothers me, but I can't put my finger on it.

Comment (bdunne) [inline app/models/branch.rb]: Naming is hard :) I'm open to suggestions

**Comment by bdunne** [comment]
@Fryguy Updated to fix the migration

**Comment by bdunne** [comment]
@Fryguy renamed ```merge_head``` to ```merge_target``` as discussed.

**Comment by chrisarcand** [comment]
Nice :+1:

**Pull request merged**